components/list/list-header.js

```diff
@@ -15,12 +15,10 @@ class ListHeader extends RtlMixin(LocalizeCoreElement(LitElement)) {
 	static get properties() {
 		return {
 			/**
-			 * @ignore
-			 * Whether to render a header with reduced whitespace
-			 * TODO: Remove
+			 * Whether to render select-all and selection summary
 			 * @type {boolean}
 			 */
-			slim: { reflect: true, type: Boolean },
+			noSelection: { type: Boolean, attribute: 'no-selection' },
```

Comment:
Should default to `false` in the constructor.

Reply:
Ah yeah, thanks.